components/message_submit_error.jsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {isErrorInvalidSlashCommand} from 'utils/post_utils.jsx';
+
+class MessageSubmitError extends React.PureComponent {
+    static propTypes = {
+        error: PropTypes.object.isRequired,
+        handleSubmit: PropTypes.func.isRequired,
+        submittedMessage: PropTypes.string,
+    }
+
+    renderSlashCommandError = () => {
+        if (!this.props.submittedMessage) {
+            return this.props.error.message;
+        }
+
+        const command = this.props.submittedMessage.split(' ')[0];
+        return (
+            <FormattedMessage
+                id='message_submit_error.invalidCommand'
```

Comment:
The key here does not match the change in `en.json`.

Reply:
Yes, my apologies for these